src/Paramore.Brighter/MessageCloudEvents.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+namespace Paramore.Brighter
+{
+    public class MessageCloudEvents
```

Comment:
Just create new MessageHeader telemetry properties

Reply:
I'll move this into headers and rename it to Telemetry, this got created early on and got the naming from the distributed Tracing stuff

Reply:
Renamed and Moved

Reply:
Yeah, this looks much better